lib/services/history/history_manager.ts

```diff
@@ -141,10 +163,20 @@ export class SNHistoryManager extends PureService {
     }
   }
 
-  historyForItem(item: SNItem) {
+  sessionHistoryForItem(item: SNItem) {
     return this.historySession!.historyForItem(item.uuid);
   }
 
+  async fetchHistoryFromServer(item: SNItem) {
+    const itemRevisionsResponse = await this.apiService!.getItemRevisions(item.uuid);
+    this.historyServer = await ServerHistoryRevision.FromResponse(this.protocolService!, item.uuid, itemRevisionsResponse);
```

Comment:
I had to pass the item's `uuid` in order to decrypt revisions. In my opinion, the server response should be the following:

- `uuid`: the `uuid` of the item
- `revision_uuid`: the `uuid` of the revision

Reply:
Why would we overwrite `this.historyServer` with item-specific data every time we fetch a new item's revision? This is overly extracted from the session history design. These two are not the same and should not be treated as such.